vnext/Microsoft.ReactNative/IReactDispatcher.idl

```diff
@@ -29,16 +29,16 @@ One primary dispatcher that applications may require is the @IReactContext.UIDis
   [webhosthidden]
   static runtimeclass ReactDispatcherHelper
   {
-    // Creates a new serial dispatcher that uses thread pool to run tasks.
+    DOC_STRING("Creates a new serial dispatcher that uses thread pool to run tasks.")
     static IReactDispatcher CreateSerialDispatcher();
 
-    // Get or create IReactDispatcher for the current UI thread.
+    DOC_STRING("Get or creates a @IReactDispatcher for the current UI thread. This can be used with @ReactInstanceSettings.UIDispatcher to launch a React instance from a non-UI thread.  This API must be called from a UI thread.  It will return null if called from a non UI thread.")
```

Comment:
```suggestion
    DOC_STRING("Get or creates a @IReactDispatcher for the current UI thread. This can be used with @ReactInstanceSettings.UIDispatcher to launch a React instance from a non-UI thread.  This API must be called from a UI thread.  It will return null if called from a non-UI thread.")
```